cmake/googletest-download/CMakeLists.txt.in

```diff
@@ -6,8 +6,8 @@ include(ExternalProject)
 ExternalProject_Add(googletest
   GIT_REPOSITORY    https://github.com/google/googletest.git
   GIT_TAG           release-1.10.0
-  SOURCE_DIR        "${CMAKE_CURRENT_BINARY_DIR}/googletest-src"
-  BINARY_DIR        "${CMAKE_CURRENT_BINARY_DIR}/googletest-build"
+  SOURCE_DIR        "${FPRIME_FRAMEWORK_PATH}/gtest/googletest-src"
```

Comment:
You may want to do `gtest/${PLATFORM}` or `gtest-${PLATFORM}` just in case people want to cross compile their unit tests.

Reply:
Tested this, it is not required.